packages/dso-toolkit/docs/20-Fundament/05-favicon.md

```diff
@@ -1,7 +1,15 @@
----
-label: Favicon
----
-| Type | Dimensions | Afbeelding                                         |
-| ---- | ---------- | -------------------------------------------------- |
-| ICO  | 16x16      | ![Favicon ICO]({{ '/images/favicon.ico' | path }}) |
-| PNG  | 32x32      | ![Favicon PNG]({{ '/images/favicon.png' | path }}) |
+## Installatie instructies
+
+Download het [zip bestand]({{ '/images/favicon/dso-favicon.zip' | path }}) en zet de inhoud in de root van de website.
+
+Voeg de volgende code toe in de `head` sectie van de pagina's:
+
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
```

Comment:
Wel backticken om 'code' te laten zien, toch?

Reply:
Nee is niet nodig, hij tovert er vanzelf een <code> blok bij:
![image](https://user-images.githubusercontent.com/5871330/116399308-138c0100-a829-11eb-9743-c89316a6256a.png)

Reply:
Wat heeft `link rel="manifest"` met deze change te maken?

Reply:
Ah ik zie het al, die zit in die zip.

```{
    "name": "Omgevingsloket",
    "short_name": "Omgevingsloket",
    "icons": [
        {
            "src": "/android-chrome-192x192.png",
            "sizes": "192x192",
            "type": "image/png"
        },
        {
            "src": "/android-chrome-512x512.png",
            "sizes": "512x512",
            "type": "image/png"
        }
    ],
    "theme_color": "#ffffff",
    "background_color": "#ffffff"
}
```